packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-account-identifier-message/cashier-onboarding-account-identifier-message.tsx

```diff
@@ -16,7 +16,7 @@ const CashierOnboardingAccountIdentifierMessage: React.FC = observer(() => {
             message={
                 <Localize
                     i18n_default_text='This is your <0>{{currency}}</0> account {{loginid}}.'
-                    values={{ currency: currency_config.display_code, loginid }}
+                    values={{ currency: currency_config?.display_code, loginid }}
```

Comment:
how is this related with `p2p_settings`?

Reply:
for some reason i got an error related to the currency config that it was undefined, so I had to do this to not get the error, but i'll check if i remove these changes, if its still there

Reply:
I just checked, I still need to leave it there, for some reason, the currency_config values are returning undefined.